packages/magento-cart-shipping-address/components/ShippingAddressForm/ShippingAddressForm.tsx

```diff
@@ -111,41 +106,23 @@ export const ShippingAddressForm = React.memo<ShippingAddressFormProps>((props)
       }
     },
   })
-  const { handleSubmit, valid, formState, required, error } = form
+  const { handleSubmit } = form
   const submit = handleSubmit(() => {})
 
   useFormPersist({ form, name: 'ShippingAddressForm' })
   useFormCompose({ form, step, submit, key: 'ShippingAddressForm' })
 
-  const autoSubmitting = useFormAutoSubmit({
-    form,
-    submit,
-    fields: ['postcode', 'countryCode', 'regionId'],
-  })
-  const readOnly = formState.isSubmitting && !autoSubmitting
-
   return (
-    <Form onSubmit={submit} noValidate sx={sx}>
-      <NameFields form={form} key='name' readOnly={readOnly} />
-      <AddressFields form={form} key='addressfields' readOnly={readOnly} />
-      <FormRow key='telephone'>
-        <TextFieldElement
-          control={form.control}
-          name='telephone'
-          variant='outlined'
-          type='text'
-          required={required.telephone}
-          validation={{
-            pattern: { value: phonePattern, message: i18n._(/* i18n */ 'Invalid phone number') },
-          }}
-          label={<Trans id='Telephone' />}
-          InputProps={{
-            readOnly,
-            endAdornment: <InputCheckmark show={valid.telephone} />,
-          }}
-        />
-      </FormRow>
-      <ApolloCartErrorAlert error={error} />
-    </Form>
+    <FormProvider {...form}>
+      {children ?? (
+        <>
+          <NameFields />
+          <AddressFields />
+          <TelephoneField />
+          <ApolloCartErrorAlert />
+          <FormAutoSubmit name={['postcode', 'countryCode', 'regionId']} />
+        </>
+      )}
+    </FormProvider>
```

Comment:
Please re-add form element (with sx) to prevent breaking change.

Reply:
Fixed